solana-programs/anchor/audius-data/cli/main.ts

```diff
@@ -346,8 +352,15 @@ const main = async () => {
       break;
     case functionTypes.initContentNode:
       console.log(`Initializing content node`)
+      const audiusLibs = await initAudiusLibs(true)
+      const ethWeb3 = audiusLibs.ethWeb3Manager.getWeb3()
+      const ethAccounts = await ethWeb3.eth.getAccounts()
       // TODO - This authority should be a delegate private key propagated from local env or passed in
-      const contentNodeAuthority = anchor.web3.Keypair.generate();
+      const delegateWallet = ethAccounts[parseInt(options.cnSpId)]
+      const ganacheEthAccounts = await getEthContractAccounts()
+      const delegatePrivateKey = ganacheEthAccounts.private_keys[String(delegateWallet)] // TODO why does this not work...
+      console.log(delegateWallet, 'AAAAAAAAAAAAA', Object.keys(ganacheEthAccounts.private_keys), delegatePrivateKey)
```

Comment:
log shows that `delegateWallet` is DEFINITELY one of the keys on that `ganacheEthAccounts.private_keys` object... however, `delegatePrivateKey` ends up `undefined` here. I must be missing something, or this is my 11:30 brain having been stuck on this awhile. Going to follow up on this tmr.

Reply:
this is bc it was formatted like a [checksum](https://coincodex.com/article/2078/ethereum-address-checksum-explained/) and therefore capitalization was off. working implementation in PR - thanks @hareeshnagaraj and @jowlee for the conversion help